pkgs/development/python-modules/Pygments/default.nix

```diff
@@ -19,7 +19,7 @@ buildPythonPackage rec {
   doCheck = false;
 
   meta = {
-    homepage = http://pygments.org/;
+    homepage = https://pygments.org/;
```

Comment:
Might as well quote these while we’re here, per NixOS/rfcs#45.

Reply:
If i knew about this beforehand, i would have. Didn't realize it was accepted yet

Reply:
the quoting can also happen automatically by script, I'm not too concerned about it.